### PR TITLE
Added units to imp.mitplasma

### PR DESCRIPTION
**Pull request by abotiamnot**

### CODE REVIEW

File: heliopy/data/helper.py [hunk 1]
```diff
@@ -145,19 +145,26 @@ def cdf_dict(unit_string):
...
                         ('#/{cc*(cm/s)^3}', (u.cm**3 * (u.cm / u.s)**3)**-1),
                         ('sec', u.s),
                         ('nT GSE', u.nT),
                         ('nT GSM', u.nT),
                         ('nT DSL', u.nT),
                         ('nT SSL', u.nT),
                         ('msec', u.ms),
-                        ('ionic charge', ionic_charge)])
+                        ('ionic charge', ionic_charge),
+                        ('Spacecraft', False)])
```
Comment (dstansby) [inline heliopy/data/helper.py]: Why `False` and not `dimensionless_unscaled`?

Comment (abotiamnot) [reply heliopy/data/helper.py]: mitplasma function isn't using Spacecraft value in the DataFrame columns so I decided it'd be better if we ignore it.

Comment (dstansby) [reply heliopy/data/helper.py]: Hmm, it probably should. Could you fix that, and change this to `dimensionless_unscaled`?

File: heliopy/data/util.py [hunk 2]
```diff
@@ -248,6 +248,8 @@ def cdf_units(cdf_, keys=None, manual_units=None):
                 message = "The CDF provided units ({}) for key '{}' \
                 are unknown".format(unknown_unit, key)
                 warnings.warn(message, Warning)
+            if temp_unit is False:
```
Comment (dstansby) [inline heliopy/data/util.py]: I think these two lines can be deleted, otherwise this looks 👍

**Pull request merged**